relay/pkg/block/push.go

```diff
@@ -212,3 +309,23 @@ func packHeaders(headers []*btc.Header) []byte {
 
 	return packed
 }
+
+func splitBatch(headers []*btc.Header, startMod int64) (
+	preChangeHeaders,
+	postChangeHeaders []*btc.Header,
+) {
+	for _, header := range headers {
+		if header.Height%difficultyEpochDuration >= startMod {
+			preChangeHeaders = append(preChangeHeaders, header)
+		} else if header.Height%difficultyEpochDuration < startMod {
+			postChangeHeaders = append(postChangeHeaders, header)
+		} else {
+			logger.Warnf(
+				"could not assign header [%v] to pre/post-change part",
+				header,
+			)
```

Comment:
Is it a warning or an error? It should never happen. I'd also log `startMod`.

Reply:
Ok. See https://github.com/keep-network/tbtc/commit/3f435f44610bcb4d040ac5ea88f57f2d070e3ad6